iothub/service/src/DeviceStreamResponse.cs

```diff
@@ -42,7 +42,7 @@ public DeviceStreamResponse(string streamName, bool isAccepted, string authoriza
             StreamName = streamName;
             IsAccepted = isAccepted;
             AuthorizationToken = authorizationToken;
-            Url = streamingGatewayUri;
+            Uri = streamingGatewayUri;
         }
```

Comment:
So are these just naming changes?

Reply:
URL becomes URI, which is a breaking change. But it is more accurate

Reply:
Did you read the bugs?

Reply:
To expand on this, there is a getter and setter for this URI, so customer's could see this break their code. I stand by it being worthwile though as the field is a URI, not a URL